R/annotateProteinInfoFromIndra.R

```diff
@@ -0,0 +1,180 @@
+#' Annotate Protein Information from Indra
+#'
+#' This function annotates a data frame with protein information from Indra.
+#'
+#' @param df A data frame containing protein information.
```

Comment:
Can you point out that it ideally is the output of groupComparison, similar to [here](https://github.com/Vitek-Lab/MSstatsBioNet/blob/devel/R/getSubnetworkFromIndra.R#L6-L8)

Reply:
done